lom/src/lib/lomcommon/config.go

```diff
@@ -494,3 +509,22 @@ func InitConfigMgr(p *ConfigFiles_t) (*ConfigMgr_t, error) {
 }
 
 
+func InitConfigPath(path string) error {
+    cfgPath := path
+    if len(path) == 0 {
+        if p, err := os.Getwd(); err != nil {
+            return LogError("Failed to get current working dir (%v)", err)
+        } else {
+            cfgPath = p
+        }
+    }
+    cfgFiles := &ConfigFiles_t {
+        GlobalFl: filepath.Join(cfgPath, "globals.conf.json"),
```

Comment:
Is it better to have underscore in name ? 
globals_conf.json 
actions_conf.json
bindings_conf.json ?

Reply:
does not matter, really.